packages/libraries/cdk/src/components/surface/ocdk-surface.ts

```diff
@@ -246,6 +246,15 @@ export class OcdkSurface extends HTMLElement {
     this.controller.setOriginCorner(originCorner);
   }
 
+  /**
+   * Check if a click is outside of the surface
+   * @param event - event reference
+   */
+  isClickOutsideSurface(event: Event): boolean {
```

Comment:
We need to discuss if that test should be done directly here, in the surface, or should be done in controller for every component using CDK